src/rt/sched/multimessage.h

```diff
@@ -19,7 +19,7 @@ namespace verona::rt
       size_t index;
       size_t count;
       Cown** cowns;
-      Action* action;
+      Behaviour* be;
```

Comment:
Do we have to shorten it to `be`? It's harder to read like that. I'd leave as `behaviour`.

Reply:
fixed